tfjs-tfdf/src/tfdf_web_api_client.js

```diff
@@ -22,4 +22,17 @@
 // imported correctly by typescript code.
 const YggdrasilDecisionForests = require('../wasm/inference.js');
 
-exports.tfdfWeb = YggdrasilDecisionForests();
+let tfdfWeb = null;
+
+// Function is used here instead of variable to make loading lazy in case user
+// uses setLocateFile before module is used.
+exports.tfdfWeb = () => {
+  if (tfdfWeb == null) {
+    tfdfWeb = YggdrasilDecisionForests();
+  }
```

Comment:
Does TFDF have state? If so, could a user ever want multiple instances of it?

Reply:
No